lib/opentelemetry-bridge/OTelSpan.js

```diff
@@ -167,6 +167,14 @@ class OTelSpan {
     return this;
   }
 
+  addLink(link) {
```

Comment:
According to the [Span interface](https://github.com/open-telemetry/opentelemetry-js/blob/main/api/src/trace/span.ts#L80-L98) both `addLink` and `addLinks` must return the span itself. You already did it in [lib/opentelemetry-bridge/OTelBridgeNonRecordingSpan.js](https://github.com/elastic/apm-agent-nodejs/pull/4078/files#diff-d05e9b0314bfa0a1928fe1612ae124355bcbc74005f626adee5880ea4cdf90f2) so why not here?

Reply:
Fixed in commit c0340b24, along with a test for it.